frontend/test/metabase/scenarios/reference/metrics.cy.spec.js

```diff
@@ -0,0 +1,82 @@
+import { signInAsAdmin, restore } from "__support__/cypress";
+import { SAMPLE_DATASET } from "__support__/cypress_sample_dataset";
+
+const { ORDERS, ORDERS_ID } = SAMPLE_DATASET;
+
+describe("scenarios > reference > metrics", () => {
+  const METRIC_NAME = "orders < 100";
+  const METRIC_DESCRIPTION = "Count of orders with a total under $100.";
+
+  beforeEach(() => {
+    restore();
+    signInAsAdmin();
+
+    cy.request("POST", "/api/metric", {
+      definition: {
+        aggregation: ["count"],
+        filter: ["<", ["field", ORDERS.TOTAL, null], 100],
+        "source-table": ORDERS_ID,
+      },
+      name: METRIC_NAME,
+      description: METRIC_DESCRIPTION,
+      table_id: ORDERS_ID,
+    });
+  });
+
+  it("should see the listing", () => {
+    cy.visit("/reference/metrics");
+    cy.contains(METRIC_NAME);
+    cy.contains(METRIC_DESCRIPTION);
```

Comment:
Ideally, `contains` would be `findByText`. It does an exact match rather than substring match. We use [Cypress testing library](https://testing-library.com/docs/cypress-testing-library/intro/) for this.